ang/civicase/ActivityCard.js

```diff
@@ -24,16 +24,17 @@
         bulkAllowed: '=',
         editActivityUrl: '=?editActivityUrl',
         type: '=type',
-        feedViewFn: '=feedViewFn'
+        viewActivityInFeed: '=?'
```

Comment:
As discussed, the popup should not be the default behaviour. only when `viewActivityInPopup` is sent, we should call that function.
We should always send the emit.
But incase the `viewActivityInPopup` is true. We call function to open the popup.